src/libs/export.c

```diff
@@ -1303,6 +1303,32 @@ void gui_init(dt_lib_module_t *self)
   dt_bauhaus_combobox_add(d->intent, _("absolute colorimetric"));
   gtk_box_pack_start(GTK_BOX(self->widget), d->intent, FALSE, TRUE, 0);
 
+  tooltip = g_strdup_printf(_("• perceptual : "
+                              "smoothly moves out-of-gamut colors into gamut,"
+                              "preserving gradations, but distorts in-gamut colors in the process."
+                              "\n\n"
+
+                              "• relative colorimetric : "
+                              "reproduces colors with as little desaturation as possible."
```

Comment:
relative colorimetric keeps luminance while reducing saturation until colors fit in gamut

Reply:
Ok, I took this as-is, simpler and less verbose.